bgpd/bgp_ecommunity.h

```diff
@@ -202,4 +202,13 @@ extern void bgp_remove_ecomm_from_aggregate_hash(
 					struct ecommunity *ecommunity);
 extern void bgp_aggr_ecommunity_remove(void *arg);
 
+
+static inline int ecommunity_strip_rts(struct ecommunity *ecom)
```

Comment:
Is this `int` type needed? I don't see return is checked anywhere in the code.

Reply:
I felt that there might be future actions based on whether RTs have been stripped or not. Shall I leave it as it is or you recommend changing?

Reply:
I changed to void - if a need materializes, it can be changed to int then